test/integration/Claims/eth-payouts.js

```diff
@@ -87,6 +86,61 @@ describe('send claim payout to the payout address', function () {
     assert(actualPayout.eq(expectedPayout), 'should have transfered the cover amount');
   });
 
+  it('[A1, status: 0, 7, 14] partial claim, CA accept, closed with closeClaim()', async function () {
+
+    const { cd, cl, qd, cr, tc } = this.contracts;
+    const coverAmount = 5;
+    const requestedAmount = 3;
+    const cover = { ...coverTemplate, amount: coverAmount };
+
+    await buyCover({ ...this.contracts, cover, coverHolder });
+    const [coverId] = await qd.getAllCoversOfUser(coverHolder);
+
+    await expectRevert(
```

Comment:
would expect this as a separate test case normally but it's acceptable since this code is sunset